### ISSUE TITLE
I think the ignore accents add-on is breaking a search feature in 2.1.29

### ISSUE THREAD

**Issue by mnhende2** [hssm/anki-ignore-accents#1]
@Hssm:  Greetings.  As always, thanks for maintaining this add-on.  It is one that I use extensively and have ever since you made it.

Anyhow, I have been running 2.1.16 for months because newer versions broke another add-on I use.  I'm just now jumping to 2.1.29 and finding a lot of other add-ons are having problems also.  I'm trying to isolate the issues.  All that to say, I hope this report is accurate. 

It appears that the ignore accents add-on is breaking "not / -" searches.  For example, if I type "is:due" I get all due cards, but normally when I type "-is:due" I get all non-due cards.  But when I run the ignore access add-on, I get no results for all negative searches.  When I disable the add-on and restart Anki, then negative searches work again.  Toggle the add-on back on and the "-" negated variable searches don't work again.

**Comment by hssm** [comment]
If you surround the search term (without the add-on) with quotes, like `"-is:due"`, it stops working. That's what the add-on is doing internally. It might be a bug in the parser (since I expect this to work) or it could just be undefined behavior which would need to be supported additionally.

For the time being, you can use `-("is:due")` to get your results. No idea why it works, it was a clever guess.

**Comment by mnhende2** [comment]
@hssm:  Thanks for the clever workaround.  I'll use that for the time being, and wish you well in trying to find solutions that can be build into the addon.

**Comment by hssm** [comment]
According to the [docs](https://docs.ankiweb.net/#/searching), the proper way to negate quoted terms is to put the `-` on the outside, so I've modified the add-on to do that instead: `-"is:due"`. I've pushed a new version to AnkiWeb with the fix.

However, looking at the documentation it seems there are more terms that this add-on breaks, notably the `or` and the `()`s. I may look at those in the future, but for now this will do.
